packages/ia-components/sandbox/languages/Languages.jsx

```diff
@@ -28,8 +27,16 @@ const languageConfig = { // Note the flags are dragged out of the Mac Emoji and
 }
 if (!currentISO()) currentISO("en");
 
+/**
+ * Fetch a supported language
+ * @param lang
+ * @param cb(err)
+ */
 function getLanguage(lang, cb) {
-  if (!languageConfig[lang]) { cb(new Error('Do not support language: '+lang));
+  if (!languageConfig[lang]) {
+    cb(new Error('Do not support language: ' + lang));
```

Comment:
does the new Error break DOM drawing?

Reply:
getLanguage is called much earlier - during language switching, not during DOM drawing